clients/java/src/main/java/io/camunda/client/impl/oauth/OAuthCredentialsProvider.java

```diff
@@ -243,30 +237,8 @@ private void maybeConfigureCustomSSLContext(final HttpURLConnection connection)
   }
 
   private SSLSocketFactory createSSLContext() {
```

Comment:
🔧 I think this method doesn't make sense anymore right ?

Reply:
Done